### DISCUSSION TITLE
DbStat table

### DISCUSSION THREAD

**Discussion by ronnyremesnik** [sqlitebrowser/sqlitebrowser#3610] [General]
Hello !
Do we have any build with the DBStat table which can show the table size ?
https://www.sqlite.org/dbstat.html
Thanks !

**Comment by justinclift** [comment]
It looks like that needs the `SQLITE_ENABLE_DBSTAT_VTAB` compile time option added:

https://www.sqlite.org/compile.html#enable_dbstat_vtab

Probably not hard to add though (@lucydodo?).

Which OS are you using? :smile:

**Reply by lucydodo** [reply]
Yes, that shouldn't be too hard.
Let me know what operating system @ronnyremesnik using, and I'll prepare a test build.

**Reply by cldnlbr** [reply]
Hi! This has been a while, but can we enable it for Windows builds?

**Reply by tbussmann** [reply]
I'd love to see that enabled for the macOS builds, as well.
